box/5.1/modules/ROOT/pages/box-connector-config-topics.adoc

```diff
@@ -38,6 +38,115 @@ Keeping these values in a separate file is useful if you need to deploy to diffe
 +
 . Click *OK*.
 
+[[jwt]]
+== JWT Authentication
+
+Authenticate your application directly to Box using a JSON Web Token (JWT).
+
+The following steps describe how to configure Box Connector using JWT:
+
+* <<create-an-app>>
+* <<generate-a-keypair>>
+* <<export-and-add>>
+
+
+[[create-an-app]]
+=== Create an App
+
+. Go to the https://app.box.com/developers/console[Developer Console].
+. Select *Create New App*.
+. Select *Custom App* for the type of application.
+. Select *Server Authentication (with JWT)* for the type of authentication.
+. Provide a name for the application.
+. Click *Create App*.
+
+image::box-custom-app-select.png[Custom app selection]
+
+
+[[generate-a-keypair]]
+=== Generate a Keypair
+
+. Download https://keystore-explorer.org[KeyStore Explorer].
+. Open the application and select *Create a new KeyStore*.
+. Select *JKS*.
+. Click *OK*.
+. Click *Save*.
+
++
+image::keystore-jks.png[Create new JKS KeyStore selection]
++
+
+. Click on *Tools* and select *Generate Key Pair*.
+. Select *RSA* with a key size of 2048.
+
++
+image::rsa-2048.png[RSA with a key size of 2048 selection]
++
+
+. Click *OK* to prompt the *Generate Key Pair Certificate* dialog box to appear.
+. Accept the default settings.
+
++
+image::generate-keypair-certificate.png[Generate Key Pair Certificate dialog box]
++
+
+. Click on the *Name* icon and add the following information:
+.. *Common Name (CN)*: mulesoft
+.. *Organization Unit (OU)*: mulesoft
+.. *Organization Name (O)*: mulesoft
+.. *Locality Name (L)*: mulesoft
+.. *State Name (ST)*: mulesoft
+.. *Country (C)*: CA
+. Click *OK*.
```

Comment:
```suggestion
* *Common Name (CN)*: mulesoft
* *Organization Unit (OU)*: mulesoft
* *Organization Name (O)*: mulesoft
* *Locality Name (L)*: mulesoft
* *State Name (ST)*: mulesoft
* *Country (C)*: CA
. Click *OK*.
```

Reply:
This is more just a styling comment, the double period would create a letter list a, b, c, d. Feel free to update it or not. Looks good